src/server.c

```diff
@@ -6859,7 +6860,11 @@ __attribute__((weak)) int main(int argc, char **argv) {
 #endif
     tzset(); /* Populates 'timezone' global. */
     zmalloc_set_oom_handler(serverOutOfMemoryHandler);
-
+#if defined(HAVE_DEFRAG)
+    // we assume jemalloc version in use supports defragmentation api
```

Comment:
Why do you say assume? We have checked the jemalloc version so we are sure here. Also, let's not assume this will always be jemalloc. There's already another PR that want's to use defrag for non-jemalloc: #1303.

I think we can delete this comment.

```suggestion
```